models/staging/tpdm/base/base_tpdm__evaluation_element.sql

```diff
@@ -0,0 +1,28 @@
+with educator_prep_programs as (
+    {{ source_edfi3('evaluation_element') }}
+),
+renamed as (
+    select
+        tenant_code,
+        api_year,
+        pull_timestamp,
+        last_modified_timestamp,
+        file_row_number,
+        filename,
+        is_deleted,
+
+        v:id::string                                                 as record_guid,
+        v:evaluationElementTitle::string                             as evaluation_element_title,
+        v:sortOrder::int                                             as sort_order,
+        v:minRating::decimal(6,3)                                    as min_rating,
+        v:maxRating::decimal(6,3)                                    as max_rating,
+        v:evaluationObjectiveReference:educationOrganizationId ::int as ed_org_id,
```

Comment:
```suggestion
        v:evaluationObjectiveReference:educationOrganizationId::int as ed_org_id,
```

Reply:
Place id's on the first places in CTE's, please. Check this pattern across all models you created